package.json

```diff
@@ -6,58 +6,31 @@
 		"checkFormat": "npm run prettier -- --list-different",
 		"coveralls": "node node_modules/coveralls/bin/coveralls.js < coverage/lcov.info",
 		"format": "npm run prettier -- --write",
-		"jest": "cross-env NODE_ENV=test jest",
 		"lerna": "lerna bootstrap -- --no-optional --no-package-lock",
 		"link": "lerna run link",
 		"lint": "eslint packages/**/*.js examples/**/*.js scripts/**/*.js",
 		"pa11y": "pa11y-ci ./packages/clay-*/demos/a11y.html",
 		"prettier": "prettier-eslint packages/**/**/*.js examples/**/**/*.js scripts/**/**/*.js",
-		"start": "http-server . -p 4000",
```

Comment:
hey @bryceosterhaus, I know we had talked about the demos and that this would not be the best option, I've been thinking about testing https://storybook.js.org/, I've seen some use cases in a monorepo, I think it is a great experience. We can test it once we start having some components in the master.

Reply:
Yeah I agree we definitely need easily accessible demos on the repo level. Storybook is pretty great, I have used it in the past and it was easy to use. One other thing I am investigating is the possibility of using "Framer X" and that could also possible be a way to run demos. I am working with some of the UX leads here in LA and gonna determine if framer fits our needs.

I think this can be a tracked in a differ issue though, so I have created one https://github.com/liferay/clay/issues/1562

Reply:
Oh great, I tested a component of Metal.js at the time with `Framer X` worked up, even though it was not React, I built a layer to give compatibility... The big problem was the CSS that are scoped.